src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -471,101 +485,100 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
+- *Attore Principale*: amministratore pubblico;
+- *Precondizioni*: il sistema è operativo e accessibile;
+- *Postcondizioni*: l'amministratore pubblico visualizza un #glossary("pannello") contenente una tabella, la quale mostra tutti i dati superanti le soglie impostate nel sistema, mostrando il valore superante la soglia, il sensore che ha rilevato tale valore e il timestamp relativo;
 - *Scenario Principale*:
-  + l'amministratore pubblico seleziona la visualizzazione della deshboard delle anomalie.
+  + L'amministratore pubblico seleziona la visualizzazione della #glossary[dashboard] delle anomalie.
 
 #figure(
   image("assets/UML/UC4.2_Visualizzazione-tabella-superamento-soglie.png",width:70%),
   caption: [UC4.2 Visualizzazione tabella superamento soglie]
 )
 
 /*TODO: per gli UC5.X andare a capire se usano soglie fisse o tramite variabili per gli allert (sempre se possibile)*/
-=== UC5: Visualizzazione allerte superamento soglie
-- *Attore Principale*: amministratore pubblico.
-- *Precondizioni*: nessuna.
-- *Postcondizioni*: l'amministratore pubblico riceve una notifica di superamento di una soglia impostata.
+=== UC5: Visualizzazione allerte superamento soglia
+- *Attore Principale*: amministratore pubblico;
+- *Precondizioni*: il sistema è operativo e accessibile;
+- *Postcondizioni*: l'amministratore pubblico riceve una notifica di superamento di una soglia impostata;
 - *Scenario Principale*:
-  + il sistema rileva condizioni che richiedono l'invio di una notifica per segnalare il superamento di una soglia impostata.
+  + Il sistema rileva condizioni che richiedono l'invio di una notifica per segnalare il superamento di una soglia impostata.
 #figure(
   image("assets/UML/UC5_Visualizzazione-allerte-superamento-soglie.png",width:70%),
   caption: [UC5 Visualizzazione allerte superamento soglie]
 )
 
 /*TODO: andare a fare l'immagine apposta*/
 #pagebreak()
-=== UC5.1: Visualizzazione allerte superamento soglia temperatura
-- *Attore Principale*: amministratore pubblico.
-- *Precondizioni*: nessuna.
-- *Postcondizioni*: l'amministratore pubblico riceve un'allerta relativa alla temperatura che notifica il superamento dei 40° celsius.
+=== UC5.1: Visualizzazione allerta superamento soglia temperatura
+- *Attore Principale*: amministratore pubblico;
+- *Precondizioni*: il sistema è operativo e accessibile;
+- *Postcondizioni*: l'amministratore pubblico riceve un'allerta relativa alla temperatura che notifica il superamento dei 40° Celsius (40°C).
 - *Scenario Principale*:
-  + il sistema rileva una temperatura nella media dei 5 minuti, superiore ai 40° celsius.
+  + Il sistema rileva una temperatura nella media dei 5 minuti, superiore ai 40° Celsius (40°C).
 #figure(
```

Comment:
nelle figure di 5.1-5.5 il titolo di UC5 in alto riporta soglie invece di soglia.. (scusa)

Reply:
l'uc5 doveva essere al plurale e tutti gli altri sottocasi al singolare quindi non serve più cambiare le immagini ;)